examples/example_libgui.ks

```diff
@@ -0,0 +1,61 @@
+run lib_window_number_dialog.
+run lib_window_vessel_stats.
+run lib_window_menu.
+run lib_window_akros_main_menu.
+run lib_process.
+
+// This is main file of akrOS, basic operating system developed by akrasuski1
+
+ag1 off.
+set terminal:height to 20.
+set terminal:width to 60.
+
+set list_of_windows to list(list(0,0,0,0),list(0,0,0,0)).
+
+function set_windows{
```

Comment:
All right. Although this file is very likely to be rewritten anyway to enable other screen divisions.